packages/vue/src/components/sign-in.vue

```diff
@@ -18,7 +23,10 @@ const emit = defineEmits([
 
 const passwordLabel = computed(() => translate('Password'));
 const forgotYourPasswordLink = computed(() =>
-  translate('Forgot your password? ')
+  // Support backwards compatibility for legacy key with trailing space
+  !hasTranslation('Forgot your password? ')
```

Comment:
Nice, thanks for supporting the old version of this

Reply:
Yep! We can remove this on our next major release.